modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -1210,11 +1210,16 @@ def compute_part_size(indexer, remote_part, part_idx, axis):
                 )
 
                 if item_to_distribute is not no_default:
-                    if isinstance(item_to_distribute, np.ndarray):
+                    if isinstance(item_to_distribute, (np.ndarray, pandas.Series)):
```

Comment:
There shouldn't be any kind of 1D sequences at this level
```suggestion
                    if isinstance(item_to_distribute, np.ndarray):
```